apps/web/package.json

```diff
@@ -81,6 +80,7 @@
     "mime-types": "^2.1.35",
     "next": "^12.1.6",
     "next-auth": "^4.3.4",
+    "next-collect": "canary",
```

Comment:
Is this safe to use? it's using `canary` version corresponding to an alpha version 🤔

Reply:
He's the library author @leog so I assume he knows what he's doing 🤔

Reply:
Oh I didn’t realize. To my point, shouldn’t we wait for a stable release to avoid issues @vklimontovich? I had very bad experiences when using unstable releases, but if we are OK with this, never mind me.

Reply:
Yes, it safe. At the moment the latest canary is stable. We haven't figured out release policy yet. We'll do it in next few days, and I'll send an additional PR to switch to latest stream. Until then we'll not going to publish any canaries anyway